config/src/main/java/org/springframework/security/config/web/server/ServerHttpSecurity.java

```diff
@@ -53,6 +53,7 @@
 import org.springframework.security.oauth2.client.web.server.OAuth2AuthorizationCodeGrantWebFilter;
```

Comment:
Please update Copyright header to 2019

Reply:
Done and updated the unit test in `OAuth2LoginTests`

Reply:
Awesome thanks! This looks great and it's ready to be merged. But before we merge can you please [squash](https://github.com/spring-projects/spring-security/blob/master/CONTRIBUTING.md#squash-commits) to 1 commit and update the commit message to follow this [format](https://github.com/spring-projects/spring-security/blob/master/CONTRIBUTING.md#format-commit-messages).

Reply:
All done.